Resources/Prototypes/Loadouts/loadout_groups.yml

```
@@ -607,6 +607,9 @@
   loadouts:
   - ChiefEngineerHead
   - ChiefEngineerBeret
+#ss220-loadout-chief-engineer-begin
+  - BeretCE
+#ss220-loadout-chief-engineer-end
```

```suggestion
  - BeretCE #SS220-CEBeret
```
изменение в одну строчку, + пометки, желательно. должны быть одни и те же на весь пр